### PR TITLE
fix replace armour with armor in the code.

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
fix change armour iteration to armor

# Explain why it's good for the game

better code is better
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:

code: fix change armour iteration to armor.

/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/WeaponStats.js
Comment (Zonespace27) [inline tgui/packages/tgui/interfaces/WeaponStats.js:1]: Find & replace made this file worse

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: tgui/packages/tgui/interfaces/WeaponStats.js
Comment (Zonespace27) [reply tgui/packages/tgui/interfaces/WeaponStats.js:1]: also, make your PR title, desc, and CL better. The fact that I had to check the code to know what this was supposed to do is very much not a good thing

**Comment by Huffie56** [comment]
@Zonespace27  i changed the tittle but i don't know what you mean when you say that i made the file worse...
 can you explain ? should i just close this PR?

**Comment by Zonespace27** [comment]
You lower cased a bunch of armors that shouldn't be

**Comment by Huffie56** [comment]
i messed up the code and the quickest way would be to restart from the ground up so i am closing this PR and reopening a new one. i learned a bunch of stuff doing this one so not all is lost

**Pull request closed without merging**